web-client/public/gene/api.js

```diff
@@ -363,15 +361,18 @@ let parseJaspar = function (data) {
                ) {
                    const errorString1 = "No gene information was retrieved for " + symbol.symbol + ".";
 
-                   const errorString2 = "This could have happened because either"
-                    + " GRNsight could not access the gene information from one of the source databases"
-                    + " or because no information exists for the gene in the source databases.";
+                   const errorString2 = "This could have happened because:";
+                   const errorString3 = "You can check back later to see if gene information" +
+                   " can be retrieved or submit an issue to https://github.com/dondi/GRNsight.";
```

Comment:
I would indent continued lines one level deeper to emphasize that they are continuations and not new statements (I wonder if I can make the linter flag those…), but that’s fine, no need to hold up the pull request over that.